src/source/geojson_source.js

```diff
@@ -138,7 +138,7 @@ class GeoJSONSource extends Evented implements Source {
                 maxZoom: options.clusterMaxZoom !== undefined ? options.clusterMaxZoom : this.maxzoom - 1,
                 minPoints: Math.max(2, options.clusterMinPoints || 2),
                 extent: EXTENT,
-                radius: (options.clusterRadius || 50) * scale,
+                radius: (options.clusterRadius ?? 50) * scale,
```

Comment:
Let's avoid the bleeding-edge ES syntax for now, since we don't transpile it:

```suggestion
                radius: (options.clusterRadius !== undefined ? options.clusterRadius : 50) * scale,
```